examples/mongodb/0.6/init-inventory.sh

```diff
@@ -0,0 +1,72 @@
+HOSTNAME=`hostname`
+
+echo "Waiting for MongoDB to start"
+
+sleep 10
+mongo localhost:27017/inventory <<-EOF
```

Comment:
I think this does not work
```
{
	"ok" : 0,
	"errmsg" : "This node was not started with the replSet option",
	"code" : 76,
	"codeName" : "NoReplicationEnabled"
}
```

Reply:
How did you start your MongoDB container? You should append ` --replSet rs0 --auth` to the invocation of mongod. I had added this to the compose file, but perhaps it should be done here indeed?

Reply:
If fixe cmd line pramaters are mandatory the it would be best to use
```
CMD ["mongod", "--replSet", "rs0", "--auth"]
```
as tha last line of the `Dockerfile`. But if I use parameters as you have requested the container fails to start
```
2017-10-16T12:30:37.890+0000 E REPL     [conn2] replSet initiate got NodeNotFound: No host described in new configuration 1 for replica set rs0 maps to this node while validating { _id: "rs0", members: [ { _id: 0.0, host: "e2d0b74aa0c0:27017" } ], version: 1 }
{
	"ok" : 0,
	"errmsg" : "No host described in new configuration 1 for replica set rs0 maps to this node",
	"code" : 93,
	"codeName" : "InvalidReplicaSetConfig"
}
bye
2017-10-16T12:30:37.892+0000 I -        [conn2] end connection 127.0.0.1:57236 (1 connection now open)
Initiated replica set
MongoDB shell version v3.4.9
connecting to: mongodb://localhost:27017/admin
2017-10-16T12:30:40.926+0000 I NETWORK  [thread1] connection accepted from 127.0.0.1:57240 #3 (1 connection now open)
2017-10-16T12:30:40.926+0000 I NETWORK  [conn3] received client metadata from 127.0.0.1:57240 conn3: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.4.9" }, os: { type: "Linux", name: "PRETTY_NAME="Debian GNU/Linux 8 (jessie)"", architecture: "x86_64", version: "Kernel 4.4.0-97-generic" } }
MongoDB server version: 3.4.9
2017-10-16T12:30:40.928+0000 E QUERY    [thread1] Error: couldn't add user: not master :
_getErrorWithCode@src/mongo/shell/utils.js:25:13
DB.prototype.createUser@src/mongo/shell/db.js:1292:15
@(shell):1:1
bye
2017-10-16T12:30:40.929+0000 I -        [conn3] end connection 127.0.0.1:57240 (1 connection now open)
```